src/UBC Toboggan/Assets/Scripts/FinishLine.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class FinishLine : MonoBehaviour
+{
+    public Collider2D triggerLevelEnd;
+    bool levelEndTriggered = false;
+    public GameObject cam;
+    public cameraFollow camScript;
+    public GameObject blackFade;
+    private SpriteRenderer blackRenderer;
+    float fadeAmount = 0f;
+    float fadeTime = 0.25f;
+    float fadeDelay = 0.25f;
+    bool startingFade = true;
+    float timer = 0.25f;
+    float transitionTime = 5f;
+
+
+    
+    void Start() {
+        camScript = cam.GetComponent<cameraFollow>();
+        blackRenderer = blackFade.GetComponent<SpriteRenderer>();
+    }
+
+    void Update() {
+        if (startingFade) {
+            timer -= Time.deltaTime;
+            fadeAmount = timer / fadeTime;
+            if (timer <= 0) {
+                timer = 0;
+                fadeAmount = 0f;
+                startingFade = false;
+            }
+            blackRenderer.color = new Color(1f,1f,1f, fadeAmount);
+        }
```

Comment:
Since `startingFade` is initialized to true, what is stopping the fade from starting as soon as we open the level?